extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -49,6 +49,7 @@ export class ValidationBubble {
 
     /** @private @const {!Element} */
     this.bubbleElement_ = ampdoc.win.document.createElement('div');
+    toggle(this.bubbleElement_, false);
```

Comment:
Why switch the default state of these?

Reply:
https://github.com/ampproject/amphtml/pull/17818/files/123a1fcccbd941500ad7503ba32e7815324dc9d9#diff-5e8c76b8524012b3c16c0cceb4f8d402L30

As of right now, `toggle` still uses inline display, so it would destroy the inline `display: block` style. You shouldn't use `setInitialDisplay` to toggle back to block.

So, I could either `toggle` _then_ `setInitialDisplay` which gets around the assert, or I can flip the default CSS. Doing that made more sense.